### PR TITLE
Removed systemctl for docker container image

### PR DESCRIPTION
**Pull request by shalakadharap**
# Problem Statement
Docker image was failing for systemd commands

# Design
-  Replaced systemd with exact internal command

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]:  Y
-  Confirm All CODACY errors are resolved [Y/N]:  Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/setup/openldap/base_configure_ldap.py [hunk 1]
```diff
@@ -114,12 +114,13 @@ def perform_base_config(rootpassword, forcecleanup, config_values):
...
         pwd = os.popen(cmd).read()
         pwd.replace('/','\/')
-        #restart slapd post cleanup
-        os.system('systemctl restart slapd')
         dn = 'olcDatabase={0}config,cn=config'
         BaseConfig.modify_attribute(dn, 'olcRootDN', (adminuser+'cn=config'))
         BaseConfig.modify_attribute(dn, 'olcRootPW', pwd)
         BaseConfig.modify_attribute(dn, 'olcAccess', '{0}to * by dn.base="gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth" write by self write by * read')
+        #restart slapd post cleanup
+        os.system('kill -9 $(pidof slapd)')
```
Comment (bkirunge7) [inline py-utils/src/utils/setup/openldap/base_configure_ldap.py:122]: It should be stop gracefully, -9 will force kill. use SIGTERM -15

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/base_configure_ldap.py]: Fixed

**Review by bkirunge7 — APPROVED** [review]

File: py-utils/src/utils/setup/openldap/base_configure_ldap.py [hunk 2]
```diff
@@ -114,12 +114,13 @@ def perform_base_config(rootpassword, forcecleanup, config_values):
...
         pwd = os.popen(cmd).read()
         pwd.replace('/','\/')
-        #restart slapd post cleanup
-        os.system('systemctl restart slapd')
         dn = 'olcDatabase={0}config,cn=config'
         BaseConfig.modify_attribute(dn, 'olcRootDN', (adminuser+'cn=config'))
         BaseConfig.modify_attribute(dn, 'olcRootPW', pwd)
         BaseConfig.modify_attribute(dn, 'olcAccess', '{0}to * by dn.base="gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth" write by self write by * read')
+        #restart slapd post cleanup
+        os.system('kill -15 $(pidof slapd)')
```
Comment (sachinpunadikar) [inline py-utils/src/utils/setup/openldap/base_configure_ldap.py:122]: Please check for return value , better use try block

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/base_configure_ldap.py:122]: os.system will never fail if internal command fails so try-catch not required here

Comment (shalakadharap) [reply py-utils/src/utils/setup/openldap/base_configure_ldap.py]: @sachinpunadikar Checked return code and handled error situation

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by bkirunge7 — APPROVED** [review]

**Pull request merged**